ext/rotoscope/rotoscope.c

```diff
@@ -275,6 +282,7 @@ VALUE rotoscope_stop_trace(VALUE self)
   if (rb_tracepoint_enabled_p(config->tracepoint))
   {
     rb_tracepoint_disable(config->tracepoint);
+    config->state = RS_OPEN;
```

Comment:
Yeah I was only thinking of being explicit about when we flip states, but I suppose they're relatively intuitive already with their naming.